docs/examples/use_cases/tensorflow/efficientdet/dataset/create_coco_tfrecord.py

```diff
@@ -160,37 +153,19 @@ def create_tf_example(
             category_names.append(category_index[category_id]["name"].encode("utf8"))
             area.append(object_annotations["area"])
 
-            if include_masks:
-                run_len_encoding = mask.frPyObjects(
-                    object_annotations["segmentation"], image_height, image_width
-                )
-                binary_mask = mask.decode(run_len_encoding)
-                if not object_annotations["iscrowd"]:
-                    binary_mask = np.amax(binary_mask, axis=2)
-                pil_image = PIL.Image.fromarray(binary_mask)
-                output_io = io.BytesIO()
-                pil_image.save(output_io, format="PNG")
-                encoded_mask_png.append(output_io.getvalue())
```

Comment:
Do we envision extending EfficientDet support to segmentation? If so, perhaps commenting out the mask support instead of outright deleting it would be sufficient?

Reply:
I believe commented out code works against readability, especially in some examples. I'd go for either keeping it or removing.

Reply:
In order to make EfficientDet example as small as possible, we removed everything connected to segmentation from the original code, not only from that part. If we wanted to support segmentation it would be too problematic at that point.